### PR TITLE
Switch to JNA-only

### PR DESCRIPTION
**Pull request by brainstorm**
As discussed in Discord on https://discord.com/channels/693329785769689098/694915062388883567/841447130949877780.

JNAerator has been deprecated since 2015 and has several issues, including but not limited to the support for ARM64 targets such as Jetson Nano, blocking progress on issue https://github.com/openpnp/openpnp/issues/916 and general development in github.com/CCHS-Melbourne/openpnp-kiosk.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,25 +57,25 @@
 				<configuration>
 					<source>1.8</source>
 					<target>1.8</target>
-					<mainClass>org.openpnp.capture.OpenPnpCapture</mainClass>
+<!--					<mainClass>org.openpnp.capture.OpenPnpCapture</mainClass>-->
```
Comment (brainstorm) [inline pom.xml:68]: According to IntelliJ IDEA this attribute is illegal.

File: pom.xml [hunk 2]
```diff
@@ -181,7 +181,7 @@
 							<encoding>UTF-8</encoding>
 							<consoleOutput>true</consoleOutput>
 							<failsOnError>true</failsOnError>
-							<linkXRef>false</linkXRef>
+<!--							<linkXRef>false</linkXRef>-->
```
Comment (brainstorm) [inline pom.xml:192]: ... this attribute is also illegal

File: pom.xml [hunk 3]
```diff
@@ -285,28 +285,28 @@
...
-										<versionRange>
-											[0.12,)
-										</versionRange>
-										<goals>
-											<goal>generate</goal>
-										</goals>
-									</pluginExecutionFilter>
-									<action>
-										<ignore></ignore>
+<!--										<ignore></ignore>-->
```
Comment (brainstorm) [inline pom.xml:299]: ... and this one is illegal too, that's why I commented them out, in addition to commenting out the JNAerator generation targets.

File: scripts/download-openpnp-capture.sh [hunk 4]
```diff
@@ -12,13 +12,15 @@ mkdir -p openpnp-capture
...
-mkdir -p openpnp-capture/binaries/darwin
+mkdir -p openpnp-capture/binaries/darwin-x86-64
+mkdir -p openpnp-capture/binaries/darwin-aarch64
 mkdir -p openpnp-capture/binaries/win32-x86
 mkdir -p openpnp-capture/binaries/win32-x86-64
 mkdir -p openpnp-capture/binaries/linux-x86-64
 mkdir -p openpnp-capture/binaries/linux-arm64
 
-curl -L -o openpnp-capture/binaries/darwin/libopenpnp-capture.dylib https://github.com/openpnp/openpnp-capture/releases/download/$1/libopenpnp-capture.dylib
+curl -L -o openpnp-capture/binaries/darwin-x86-64/libopenpnp-capture.dylib https://github.com/openpnp/openpnp-capture/releases/download/$1/libopenpnp-capture.dylib
```
Comment (vonnieda) [inline scripts/download-openpnp-capture.sh]: Isn't this just downloading the same binary twice? Is it a fat binary with both archs in it?

File: .gitignore [hunk 5]
```diff
@@ -1,4 +1,3 @@
-target
```
Comment (vonnieda) [inline .gitignore:1]: Why remove target?

File: .github/workflows/build.yml [hunk 6]
```diff
@@ -30,7 +30,7 @@ jobs:
       - name: Build
         run: |
           ./scripts/download-openpnp-capture.sh v0.0.22
-          mvn -B clean package
```
Comment (vonnieda) [inline .github/workflows/build.yml:33]: Removing clean is okay, but why remove -B?

File: .gitignore (hunk 5 above)
Comment (brainstorm) [reply .gitignore:1]: JNAerator generates the classes that we need right there under target (see files in this PR).

That being said, I can exclude the other auto-generated stuff from JNAerator we don't need under `target`, namely `*.class`, `*.xml`, etc...).

File: .github/workflows/build.yml (hunk 6 above)
Comment (brainstorm) [reply .github/workflows/build.yml:33]: I just... liked the colors? :-S:

>  -B,--batch-mode                        Run in non-interactive (batch)
>                                        mode (disables output color)

ASCII term colors might have been problematic in the past in some terminals, but both GitHub Actions and modern terminals take it well nowadays... happy to put it back, no worries.

File: scripts/download-openpnp-capture.sh (hunk 4 above)
Comment (brainstorm) [reply scripts/download-openpnp-capture.sh]: Yep, those were the "unintended issues" that needed a bit of review and that I anticipated happening, will fix, thx!

**Comment by brainstorm** [comment]
@vonnieda It makes sense that my last couple of commits break this build since ideally I would need https://github.com/openpnp/openpnp-capture/pull/47 merged in order to release `openpnp-capture-0.0.23` so the right assets can be pulled from here, but I reckon I've covered the issues you identified now :)

**Comment by brainstorm** [comment]
Woah, nice to see this merged at last! :D

**Pull request merged**